Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Components/PolyArtifactComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+
+/// <summary>
+/// Artifact polymorphs surrounding entities when triggered.
+/// </summary>
+[RegisterComponent]
+public sealed partial class PolyArtifactComponent : Component
+{
+    /// <summary>
+    /// range of the effect.
+    /// </summary>
+    [DataField("range")]
```

Comment:
```suggestion
    [DataField]
```

names can be inferred now